src/thunks/cleanTrashedGroups.js

```diff
@@ -18,10 +18,14 @@ const cleanTrashedGroups = () => async (dispatch, getState, { client }) => {
 const removeGroupFromAllContacts = async (client, groupId) => {
   const { data: contacts, next: hasMore } = await client.query(
     client.find(DOCTYPE_CONTACTS).where({
-      groups: {
-        $elemMatch: {
-          _id: groupId,
-          _type: DOCTYPE_CONTACT_GROUPS
+      relationships: {
+        groups: {
+          data: {
+            $elemMatch: {
+              _id: groupId,
+              _type: DOCTYPE_CONTACT_GROUPS
+            }
+          }
```

Comment:
I think this is where and why you need the use of `.includes('groups')`, nop ?

Reply:
No, I don't actually care about the groups themselves right there, I just need the ids. Either that or I misunderstood what ypu meant.